core/templates/dev/head/css/oppia.css

```diff
@@ -858,6 +858,13 @@ textarea {
     margin-top: 0;
     margin-left: 5px;
   }
+  ul.nav.navbar-nav.oppia-navbar-nav.navbar-right {
+    display: flex;
+    float: right;
+    margin: 0;
+  }
+}
+@media(max-width: 651px) {
```

Comment:
This seems to cause an issue on some explorations.

PR: ![](https://i.imgur.com/Xv7NmZ0.png)

Develop: ![](https://i.imgur.com/Ng9eqou.png)